yarn.lock

```diff
@@ -15311,7 +15310,7 @@ request@2.87.0:
     tunnel-agent "^0.6.0"
     uuid "^3.1.0"
 
-request@^2.74.0, request@^2.83.0, request@^2.87.0, request@^2.88.0:
+request@2.88.2, request@^2.74.0, request@^2.83.0, request@^2.87.0, request@^2.88.0:
```

Comment:
The yarn.lock has been changed, is it wanted?

Reply:
Yes I did a setup